From f4057dfb7648cc09599338cbb3f79d77ec5300c6 Mon Sep 17 00:00:00 2001
From: rajucm <rajkumar.mallikarjun.chegaraddi@intel.com>
Date: Tue, 20 Mar 2018 02:15:24 +0530
Subject: [PATCH 01/12] sensors: fix sensor cts issue for android aia2.0

Android AIA doesnt support sensors.
Remove permissions related to compass & accelerometer sensor

Change-Id: I8ca7587c0762218eba40ad7aa2ad2e9845022d62
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-58932
Signed-off-by: rajucm <raju.mallikarjun.chegaraddi@intel.com>
---
 data/etc/tablet_core_hardware.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/etc/tablet_core_hardware.xml b/data/etc/tablet_core_hardware.xml
index 6db2627e12..6ab8eae71e 100644
--- a/data/etc/tablet_core_hardware.xml
+++ b/data/etc/tablet_core_hardware.xml
@@ -29,8 +29,8 @@
     <feature name="android.hardware.audio.output" />
     <feature name="android.hardware.location" />
     <feature name="android.hardware.location.network" />
-    <feature name="android.hardware.sensor.compass" />
-    <feature name="android.hardware.sensor.accelerometer" />
+    <!-- feature name="android.hardware.sensor.compass" -->
+    <!-- feature name="android.hardware.sensor.accelerometer" -->
     <feature name="android.hardware.bluetooth" />
     <feature name="android.hardware.touchscreen" />
     <feature name="android.hardware.touchscreen.multitouch" />
-- 
2.17.1

